integration/osio_test.go

```diff
@@ -0,0 +1,125 @@
+package integration
+
+import (
+	"fmt"
+	"net"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/containous/traefik/integration/try"
+	"github.com/containous/traefik/log"
+	"github.com/go-check/check"
+	checker "github.com/vdemeester/shakers"
+)
+
+const (
+	witURL  = "http://127.0.0.1:9090"
+	authURL = "http://127.0.0.1:9091"
+)
+
+// AccessLogSuite
+type OSIOSuite struct{ BaseSuite }
+
+func (s *OSIOSuite) TestOSIO(c *check.C) {
```

Comment:
this test start wit, auth server.  start osio server and then make request to traefik endpoint.